apple/Elements/RNSVGSvgView.mm

```diff
@@ -293,7 +293,7 @@ - (void)drawRect:(CGRect)rect
   _boundingBox = rect;
```

Comment:
Is this the issue? Shouldn't this be set to `[self bounds]`?

Reply:
That didn't seem to have any effect in my testing.

The remaining issue I found is that SVG images will still render with a large width/height and not properly clip. Tracing the callstack , I got the issue to to this method in RNSVGGroups.mm:

```
- (void)setupGlyphContext:(CGContextRef)context
{
  CGRect clipBounds = CGContextGetClipBoundingBox(context); <----
  ....
}
```

`CGContextGetClipBoundingBox` returns an abnormally large box. My next step is to compare what it returns when compiled against Xcode 14/macOS 13

Reply:
Did you ever figure out what was going on for your case? We were seeing a similar issue where SVGImages weren't rendering properly, except for us `CGContextGetClipBoundingBox` at that point was abnormally small, with `width` and `height` of 1.

I'm not too familiar with Apple APIs, but the farthest I got was noticing that the abnormal context came from [the call to `UIGraphicsGetCurrentContext`](https://github.com/software-mansion/react-native-svg/blob/main/apple/Elements/RNSVGSvgView.mm#L294). Before I could investigate further, we suddenly started getting reasonable bounding boxes and I was no longer able to reproduce 😕.

Reply:
I did not get any further. Might be worth filing a high issue to track. I'm curious what your use case is?

Reply:
We're loading an image of a map and then drawing annotations that a user specified using SVGs, basically. 

I'll try to file another issue once we start seeing it again, but it unfortunately (fortunately?) seems to work on my machine™.

Reply:
FWIW, the method you linked is a UIKit (aka, iOS only) method we shim on macOS: https://github.com/microsoft/react-native-macos/blob/2fe53e8050a9c0b0906c0e092764ffdebcc039d4/packages/react-native/React/Base/macOS/RCTUIKit.m#L29-L32

I've been meaning to revisit some of these shims, but ultimately I think it would call the same underlying NSGraphicsContext code. I couldn't find any apple docs about  `CGContextGetClipBoundingBox` changing, but It very much feels like this method also had a breaking change with Xcode 15 / the macOS 14 SDK.